integration-examples/apiScripts/getMetricsForHost.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+#
+# This script will get all unique metrics for a given host. 
+#
+# Edit token.yaml to contain valid
+# - Access Token (access_token)
+# - Realm (realm)
+#
+# Syntax: python3 getMetricsForHost.py -h <HOST_NAME>
+# or
+#         python3 getMetricsForHost.py -h <HOST_NAME> -r <realm> -t <token>
+#
+# HOST_NAME should be an exact match
+
+import argparse
+import yaml
+import requests
+import json
+
+def run(hostname, realm, token):
+  limit = 5000
+  url = "https://api.{}.signalfx.com/v2/metrictimeseries?limit={}&query=host.name:{}".format(realm, limit, hostname)
+  headers = {"Content-Type": "application/json", "X-SF-TOKEN": "{}".format(token) }
+  response = requests.get(url, headers=headers)
+  responseJSON = json.loads(response.text)
+  #print(response.text)
```

Comment:
may be worth removing this comment? 🤷

Reply:
Totally fine to. I left it in thinking I occasionally used it, but any python dev probably knows they can do that.